eCourse_backend/file_exchange/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from django.forms.widgets import TextInput
+from django.utils.dateparse import parse_duration
+from users.models import Lecturer
+from .models import Submission
+from courses.models import Exercise
+
+# DatetTime input fields
+
+
+class SD_DateTimeInput(forms.DateTimeInput):
+    input_type = 'datetime-local'
+
+    def __init__(self, **kwargs):
+        kwargs["format"] = "%Y-%m-%dT%H:%M"
+        super().__init__(**kwargs)
+
+# duration field
+
+
+class DurationInput(TextInput):
```

Comment:
Please put it directly over the class you're trying to document. Maybe even use a docstring